wkconnect/utils/types.py

```diff
@@ -90,6 +103,9 @@ class Box3D(NamedTuple):
     def from_size(cls, left: Vec3D, size: Vec3D) -> Box3D:
         return cls(left, left + size)
 
+    def to_int(self) -> Box3D:
```

Comment:
Shouldn't `Box3D` already consist of `int`s?

Reply:
In theory yes. But there is nothing preventing the Vec3D's to have floats...